contracts/src/AttestationRegistry.sol

```diff
@@ -272,9 +272,12 @@ contract AttestationRegistry is OwnableUpgradeable {
   function balanceOf(address account, uint256 id) public view returns (uint256) {
     bytes32 attestationId = bytes32(abi.encode(id));
     Attestation memory attestation = attestations[attestationId];
-    if (attestation.subject.length > 20 && keccak256(attestation.subject) == keccak256(abi.encode(account))) return 1;
-    if (attestation.subject.length == 20 && keccak256(attestation.subject) == keccak256(abi.encodePacked(account)))
+    if (attestation.subject.length > 20 && keccak256(attestation.subject) == keccak256(abi.encode(account))) {
```

Comment:
Could both ifs managed by one condition?

attestation.subject.length >= 20 as return value is same.

Reply:
Line 275 uses `abi.encode` whereas line 278 uses `abi.encodePacked`

Reply:
`if (attestation.subject.length >= 20 && (keccak256(attestation.subject) == keccak256(abi.encode(account))) || keccak256(attestation.subject) == keccak256(abi.encodePacked(account)))`

I am happy with current syntax too. Just trying to save one if block but not sure about readability